package.json

```diff
@@ -146,12 +147,12 @@
     "dev": "next dev",
     "spellcheck": "cspell \"src/**/*.mdx\" --no-progress",
     "spellcheck-diff": "git diff --name-only --cached | awk \"/src.*\\.mdx/{print}\" | npx cspell --no-must-find-files --file-list stdin",
-    "build": "node tasks/generate-sitemap.mjs && next build && next export -o client/www/next-build",
+    "build": "node src/directory/generateDirectory.mjs && node tasks/generate-sitemap.mjs && next build && next export -o client/www/next-build",
     "build:release": "yarn build && next-image-export-optimizer --exportFolderPath client/www/next-build",
     "next-build": "next build",
     "next-start": "next start",
-    "amplify:submissionsLambda": "cd amplify/backend/function/submissionsLambda/src && yarn install && yarn build && yarn test",
     "prepare": "husky install",
-    "analyze": "ANALYZE=true yarn next-build"
+    "analyze": "ANALYZE=true yarn next-build",
+    "postinstall": "node src/directory/generateDirectory.mjs"
```

Comment:
Do we need this postinstall if it's part of build (or prebuild if you move it there)?

Reply:
Great idea, will do that, thanks Heather